src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/OperatorOverloadsHaveNamedAlternatesTests.Fixer.cs

```diff
@@ -10,24 +9,402 @@ namespace Microsoft.ApiDesignGuidelines.Analyzers.UnitTests
 {
     public class OperatorOverloadsHaveNamedAlternatesFixerTests : CodeFixTestBase
     {
+        #region Boilerplate
+
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicOperatorOverloadsHaveNamedAlternatesAnalyzer();
+            return new OperatorOverloadsHaveNamedAlternatesAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpOperatorOverloadsHaveNamedAlternatesAnalyzer();
+            return new OperatorOverloadsHaveNamedAlternatesAnalyzer();
         }
 
         protected override CodeFixProvider GetBasicCodeFixProvider()
         {
-            return new BasicOperatorOverloadsHaveNamedAlternatesFixer();
+            return new OperatorOverloadsHaveNamedAlternatesFixer();
         }
 
         protected override CodeFixProvider GetCSharpCodeFixProvider()
         {
-            return new CSharpOperatorOverloadsHaveNamedAlternatesFixer();
+            return new OperatorOverloadsHaveNamedAlternatesFixer();
+        }
+
+        #endregion
+
+        #region C# tests
+
+        [Fact]
+        public void AddAlternateMethod_CSharp()
+        {
+            VerifyCSharpFix(@"
+class C
+{
+    public static C operator +(C left, C right) { return new C(); }
+}
+",
+@"
+class C
+{
+    public static C operator +(C left, C right) { return new C(); }
+
+    public static C Add(C left, C right)
+    {
+        throw new System.NotImplementedException();
```

Comment:
I'd considered this, but ultimately decided against it because not all operators have such obvious defaults.  E.g., if this fixer generates a `ToInt32()` method, how does that `int` get generated?  Because of this I decided to stick with the known pattern of throwing.